Standards/scs-0003-v1-sovereign-cloud-standards-yaml.md

```diff
@@ -175,80 +253,63 @@ _Note_: the `executable` could in principle also be given via a URL; however, th
-  url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Tests/scs-compatible-iaas.yaml
-variables:
-  - os_cloud
-versions:
-  - version: v5 # This version is in a draft state and work in progress
-    # No stabilized_at: date set yet
-    standards:
-      - name: Flavor naming
-        url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Standards/scs-0100-v2-flavor-naming.md
-        condition: mandatory # is default and can be left out
-        checks:
-          - executable: flavor-name-check.py
-            env:
-              OS_CLOUD: "{os_cloud}"
-            id: flavor-name-check
-            lifetime: day
-      - name: Image metadata
-        url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Standards/scs-0102-v1-image-metadata.md
-        condition: mandatory
-        checks:
-          - executable: image-md-check.py
-            args: -c {os_cloud} -v
-            id: image-md-check
-            lifetime: day
-          - executable: image-md-check2.py
-            condition: optional
-            id: image-md-check-2
-            lifetime: day
-      - name: Standard images
-        url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Standards/scs-0104-v1-standard-images.md
-        parameters:
-          image_spec: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Tests/iaas/scs-0104-v1-images.yaml
-        checks:
-          - executable: ./iaas/standard-images/images-openstack.py
-            args: -c {os_cloud} -d {image_spec}
-            id: standard-images-check
-  - version: v4 # This is the upcoming version with a given target date. No further changes should be done to this set of standards
-    stabilized_at: 2022-04-01
-    standards:
-      - name: ....
-
-  - version: v3 # This is the stable set of standards that is currently active
-    stabilized_at: 2021-10-01
-    deprecated_at: 2022-11-08
-    standards:
-      - name: ....
-
-  - version: v2 # This set of standards is obsolete and has been replaced by v3
-    stabilized_at: 2021-07-01
-    deprecated_at: 2021-11-01
-    standards:
-      - name: ....
-```
+### Test-case descriptor
+
+The following fields are valid for every check descriptor:
+
+| Key               | Type            | Description                                                                                                       | Example           |
+| ----------------- | --------------- | ----------------------------------------------------------------------------------------------------------------- | ----------------- |
+| `id`              | String          | Identifier for this test case (immutable and unique within this module)                                           | `image-md-check`  |
+| `lifetime`        | String          | One of: `day` (_default_), `week`, `month`, `quarter`; the test result is valid until the end of the next period  | `week`            |
```

Comment:
Yes, this is intentional. It has been discussed.

We could say the result is valid for the duration of three months, starting from the time of the test. This would be simple enough. For some reason, we wanted it to align with the calendar. But then, what other options are there?

1. end of the current period: then you will have a gap at the beginning of the next period
2. end of the current period PLUS, say, the next smaller period, e.g., end of THIS quarter PLUS one month

I don't dislike the second option. It would be something for the SIG to discuss.